tst/utils.g

```diff
@@ -66,3 +66,27 @@ IsSubsetSU := function(n, q, G)
   Assert(0, CheckSesquilinearForm(G));
   return true;
 end;
+
+IsSubsetOmega := function(epsilon, n, q, G)
+  local field, invariantForm;
+  field := GF(q);
+  if IsEvenInt(q) then
+    Assert(0, CheckQuadraticForm(G));
+    invariantForm := QuadraticFormByMatrix(InvariantQuadraticForm(G).matrix, field);
+  else
+    Assert(0, CheckBilinearForm(G));
+    invariantForm := BilinearFormByMatrix(InvariantBilinearForm(G).matrix, field);
+  fi;
+  if epsilon = 0 then
+    Assert(0, WittIndex(invariantForm) = QuoInt(n - 1, 2));
+  elif epsilon = 1 then
+    Assert(0, WittIndex(invariantForm) = QuoInt(n, 2));
+  elif epsilon = -1 then
+    Assert(0, WittIndex(invariantForm) = QuoInt(n, 2) - 1);
+  fi;
+  Assert(0, DimensionOfMatrixGroup(G) = n);
+  Assert(0, DefaultFieldOfMatrixGroup(G) = field);
+  Assert(0, CheckGeneratorsSpecial(G));
```

Comment:
In the other functions I started with these three, as they are sorted in order of "price", i.e., the cheapest check first, etc. -- there's no point in checking the spinor norm if the matrices have the wrong dimension or are not invertible. In the latter case, that ensures slightly more helpful errors, too (e.g. don't waste time debugging the Witt index when the real issue is that the matrix was not invertible)

But again, it doesn't really matter